lib/openstudio-standards/standards/Standards.Model.rb

```diff
@@ -36,7 +36,17 @@ def model_create_prm_baseline_building(model, building_type, climate_zone, custo
 
   def model_create_prm_any_baseline_building(model, building_type, climate_zone, hvac_building_type = 'All others', wwr_building_type = 'All others', swh_building_type = 'All others', custom = nil, sizing_run_dir = Dir.pwd, debug = false)
     model.getBuilding.setName("#{template}-#{building_type}-#{climate_zone} PRM baseline created: #{Time.new}")
- 
+
+    # Perform a sizing run of the proposed model.
+    # Intend is to get individual space load to determine each space's
+    # conditioning type: conditioned, unconditioned, semiheated.
+    puts model_create_prm_baseline_building_requires_proposed_model_sizing_run(model)
```

Comment:
@yunyangye, could you please remove this line as part #811, it shouldn't be there. Thanks!